tests/test_default_gaussian.py

```diff
@@ -419,7 +419,7 @@ def test_apply_errors(self):
 
         with self.assertRaisesRegex(ValueError, "Invalid target subsystems provided in 'wires' argument"):
             p = U2
-            dev = DefaultGaussian(wires=4, shots=0, hbar=hbar)
+            dev = DefaultGaussian(wires=4, shots=1, hbar=hbar, analytic=True)

```

Comment:
This test should have continued to pass without needing to add `analytic=True`, right?